tests/unit_tests/models/test-environment.php

```diff
@@ -18,6 +18,90 @@ public function __construct() {
   }
 
   /**
+   * Ensure correct connection info for development environments
+   * Development environment connection info includes git parameters
+   *
+   * @return void
+   *
+   * @vcr site_connection-info
+   */
+  public function testConnectionInfoDev() {
+    logInWithBehatCredentials();
+    $site     = $this->sites->get('behat-tests');
+    $test_env = $site->environments->get('dev');
+    $info     = $test_env->connectionInfo();
+
+    // SFTP Connection Info
+    $sftpInfo = [
+        'sftp_username' => 'dev.11111111-1111-1111-1111-111111111111',
+        'sftp_host'     => 'appserver.dev.11111111-1111-1111-1111-111111111111.drush.in',
+        'sftp_password' => 'Use your account password',
+        'sftp_url'      => 'sftp://dev.11111111-1111-1111-1111-111111111111@appserver.dev.11111111-1111-1111-1111-111111111111.drush.in:2222',
+        'sftp_command'  => 'sftp -o Port=2222 dev.11111111-1111-1111-1111-111111111111@appserver.dev.11111111-1111-1111-1111-111111111111.drush.in',
+    ];
+    $this->assertArraySubset($sftpInfo, $info);
+
+    // Git Connection Info
+    $gitInfo = [
+        'git_username' => 'codeserver.dev.11111111-1111-1111-1111-111111111111',
+        'git_host'     => 'codeserver.dev.11111111-1111-1111-1111-111111111111.drush.in',
+        'git_port'     => '2222',
+        'git_url'      => 'ssh://codeserver.dev.11111111-1111-1111-1111-111111111111@codeserver.dev.11111111-1111-1111-1111-111111111111.drush.in:2222/~/repository.git',
+        'git_command'  => 'git clone ssh://codeserver.dev.11111111-1111-1111-1111-111111111111@codeserver.dev.11111111-1111-1111-1111-111111111111.drush.in:2222/~/repository.git behat-tests',
```

Comment:
You can take this whole ssh://... block out and save it as another string since it's getting repeated and then add it in to fix the line length.